include/Dialect/VectorExt/VectorExtOps.td

```diff
@@ -59,7 +59,7 @@ def VectorExt_YieldOp : VectorExt_Op<"yield", [
   let assemblyFormat = "attr-dict ($operands^ `:` type($operands))?";
 }
 
-def VectorExt_WarpSingleLaneOp : VectorExt_Op<"warp_execute_on_lane_0",
+def VectorExt_WarpExecuteOnLane0 : VectorExt_Op<"warp_execute_on_lane_0",
```

Comment:
Are you sure you want to drop the `Op` suffix? All ops in the vector dialect end with `Op`.

Reply:
Good catch, fixed it